Patterns/English/English-DateTime.yaml

```diff
@@ -374,13 +374,13 @@ TimeRegexWithDotConnector: !nestedRegex
   def: ({BaseDateTime.HourRegex}(\s*\.\s*){BaseDateTime.MinuteRegex})
   references: [ BaseDateTime.HourRegex, BaseDateTime.MinuteRegex ]
 TimeRegex1: !nestedRegex
-  def: \b({TimePrefix}\s+)?({WrittenTimeRegex}|{HourNumRegex}|{BaseDateTime.HourRegex})(\s*|[.]){DescRegex}
+  def: \b({TimePrefix}\s+)?({WrittenTimeRegex}|{HourNumRegex}|(?<!\d[,.]){BaseDateTime.HourRegex})(\s*|[.]){DescRegex}
   references: [ TimePrefix, WrittenTimeRegex, HourNumRegex, BaseDateTime.HourRegex, DescRegex ]
```

Comment:
We need a solution that works across all cultures. Shouldn’t perhaps an approach like this be applied at the BaseDateTime level?
Also, ideally we should also have at least a workaround for Java, if possible.

Reply:
ok, moved implementation to base code (following the solution already used in DateExtractor) and re-enabled test case in Java.